object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,37 @@
+/* exported omit */
+
+function omit(source, keys) {
+  var newObject = {};
+  var newIndex = 0;
+  var testArray = [];
+  var newKey;
+
+  for (var testKey in source) {
+    testArray.push(testKey);
+  }
```

Comment:
Yeah, you kinda went down a windy road here. You _do_ need to loop through the keys of the source, but rather than put them in their own array, check to see if they are inside of the `keys` to omit. If they are not, add them to the `newObject`.